src/libman/Contracts/Constants.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Web.LibraryManager.Tools.Contracts
+{
+    /// <summary>
+    /// Defines constants for libman.
```

Comment:
why do you need a copy of these constants. Aren't they the same for all extensions?

Reply:
They weren't public. They should be. I am planning to file follow up issues for moving duplicated/ shared code to right place. 
So as to not make this PR bigger than it already is.